app/assets/templates/header.html

```diff
@@ -11,7 +11,7 @@
           </button>
           <ul class="dropdown-menu list-inline" role="menu">
             <li class="dropdown-list" ng-repeat='category in categories'>
-              <a ng-href="#{{category.link}}" class="dropdown-category">{{ category.name }}</a>
+              <a ng-href="#/categories/{{ category.link }}" class="dropdown-category">{{ category.name }}</a>
```

Comment:
You have keen eyes!! 
We kept the jbuilder the same to minimise the html coding changes. So the jbuilder is
json.link category.simplified_name